contents/blog/gdpr-compliant-analytics.md

```diff
@@ -0,0 +1,83 @@
+* The person gave you specific, unambiguous consent to process the data. (e.g. They’ve opted in to your marketing email list.)
+* Processing is necessary to enter into a contract to someone. (e.g. You need to do a background check.)
+* You need to process it to comply with a legal obligation of yours. (e.g. You receive an order from the court in your jurisdiction.)
+* You need to process the data to save somebody’s life. (e.g. Well, you’ll probably know when this one applies.)
+* Processing is necessary to perform a task in the public interest or to carry out some official function. (e.g. You’re a private garbage collection company.)
+* You have a legitimate interest to process someone’s personal data. This is the most flexible lawful basis, though the “fundamental rights and freedoms of the data subject” always override your interests, especially if it’s a child’s data
+
+**Unambiguous Consent**
+
+There are specific rules about what consent means, hiding it away on page 73 or of your terms and conditions is not good enough:
+
+* Consent must be “freely given, specific, informed and unambiguous.”
+* Requests for consent must be “clearly distinguishable from the other matters” and presented in “clear and plain language.”
+* Data subjects can withdraw previously given consent whenever they want, and you have to honor their decision.
+* Children under 13 can only give consent with permission from their parent.
+* You need to keep documentary evidence of consent.
+
+So if you're tracking users in your product using PostHog to improve your product, you should explicitly ask for consent to use this data and explain exactly how you will use it when users sign up for your service.
+
+If you use PostHog with cookies on your website (for logged out users), you should also use a cookie banner to enable people to give and withdraw their consent for using cookies.
+
+**Security**
+
+You’re required to handle data securely by implementing “appropriate technical and organizational measures.”
+
+This means both technical measures (like encrypting data) and organizational measures (like staff training and limiting access to personal data).
+
+If you have a data breach, you have 72 hours to tell the data subjects or face penalties. (This notification requirement may be waived if you use technological safeguards, such as encryption, to render data useless to an attacker.)
+
+## How is GDPR compliance different with PostHog?
+
+PostHog is different from most other product analytics tools such as Mixpanel or Amplitude because it enables you to self-host on your own infrastructure and maintain full control of the data, this means you can decide on the technical and organizational security measures that are most appropriate for your organization. It also means that you can store your data within the EU, encouraging consent from EU citizens and making it easy to comply with any future data sovereignty requirements.
```

Comment:
```suggestion
PostHog is different from most other product analytics tools such as Mixpanel or Amplitude because you can you self-host it on your own infrastructure. You maintain full control of the data and you can decide on the technical and organizational security measures that are most appropriate for your organization. It also means that you can store your data within the EU, encouraging consent from EU citizens and making it easy to comply with any future data sovereignty requirements.
```